docs/_docs/versions.md

```diff
@@ -7,112 +7,161 @@ redirect_from: /docs/
+
+Zapier uses semantic versioning numbers, which gives you three options to choose from:
+
+* **Patch** (e.g. 1.0.0 to 1.0.1) can be used for changes that are still compatible with the previous version, such as fixing a bug or updating help text.
+* **Minor** (e.g. 1.0.0 to 1.1.0) is recommended for changes that add new functionality without changing what’s already there, such as creating a new trigger or action.
+* **Major** (e.g. 1.0.0 to 2.0.0) is used when making changes that are likely to break existing Zaps, such as removing triggers or actions, changing authentication methods, or starting over from scratch.
+
+Select the version that feels correct based on your plans, then click **Clone** to create the new version and start editing it.
+
+## Promoting a version
+
+After your integration has entered the _Beta_ or _Public_ status, one of its versions also becomes _Public_, meaning users see and use it by default. The process of changing which version is Public is called _Promotion_, and can be done from either the CLI or the Versions page. 
+
+In the CLI, running `zapier promote [version]` will make the specified version number the new Public version. [Read more about that here](https://platform.zapier.com/cli_docs/docs#promoting-an-app-version).
+
+In the Visual Builder, head to **Manage** then **Versions**, and select the **gear icon** next to the version you’d like to make public. In the menu that appears, select **Promote**.
+
+![Selecting 'Promote' on the Versions page](https://cdn.zappy.app/dd4fccc5acb2beb2194ca448871c0d4d.png)
+
+If you’re working with an integration that’s currently private, the _Promote_ option won’t appear. Instead, make sure users you’d like to test or move to the new integration version are invited to that version, and encourage them to try it out.
+
+### What happens when a version is promoted?
+
+* Zap templates are updated to use the new Public version of your app
+* Any new triggers or actions will appear on your integration’s public app page
+* When a user selects your integration in a new Zap, they’ll use the promoted version by default
+
+## Migrating users
 
 Once you’ve added all the changes needed to your integration, you can start rolling it out to users to update their existing Zaps and let users make new Zaps with your new integration features.
 
-If you made a patch version with only minor, non-breaking changes, you can migrate existing users and Zaps to your new integration version. Select the _Migrate_ option on your new integration version, then choose which older version of your integration to move users from, and the new version to move them to. Finally, choose what percentage of users to migrate. 
+If you made a new version with only minor, non-breaking changes, you can migrate existing users and Zaps to your new integration version. On the **Versions** page, select the **Migrate** option on your new integration version.
```

Comment:
We want to make sure that people are good and familiar with what constitutes a breaking change before they attempt to migrate, so maybe change to "If you made a new version with only minor, non-breaking changes(read more on breaking changes here)" and link to the header below?